### ISSUE TITLE
Series titles incorrectly capitalized

### ISSUE THREAD

**Issue by RobertGoulding** [jgm/pandoc#8209]
**Explain the problem.**
When the language field is set to a non-English value in a bibtex entry, the formatted reference should not change the capitalization in the series title. But in fact it does. In the minimal example attached, the series title should read "Quellen und Studien zur Philosophie"  (as it reads in the bib file) but instead it reads "Quellen Und Studien Zur Philosophie"

I have tried this with several Chicago .csl files, with the same result. The same csl file in my bibliography manager (Paperpile) keeps the series title capitalization correctly unaltered from how it appears in the bib file.

Process attached file with pandoc -C --bibliography=sample.bib --csl=chicago-author-date.csl -o sample.pdf sample.md

(Note I had to add .txt extension to bib and csl file in order to attach)

**Pandoc version?**
2.18, on a Chromebook (= debian bullseye)

[chicago-author-date.csl.txt](https://github.com/jgm/pandoc/files/9227800/chicago-author-date.csl.txt)
[sample.bib.txt](https://github.com/jgm/pandoc/files/9227801/sample.bib.txt)
[sample.md](https://github.com/jgm/pandoc/files/9227799/sample.md)

**Comment by jgm** [comment]
Thanks for the report. Trying to figure out where the problem lies, exactly.

If you do `pandoc sample.bib -f biblatex -t native -s` you can see how pandoc translates this to native citations:

```
                       , ( "collection-title"
                         , MetaInlines
                             [ Str "Quellen"
                             , Space
                             , Str "und"
                             , Space
                             , Str "studien"
                             , Space
                             , Str "zur"
                             , Space
                             , Str "philosophie"
                             ]
                         )
```

Interestingly, this has been changed to sentence case and the capitalization lost. So, that seems like one problem.

But perhaps there is a second problem in citeproc processing itself.

With the following citeproc test case
```
>>==== MODE ====>>
bibliography
<<==== MODE ====<<

>>==== RESULT ====>>
<<==== RESULT ====<<

>>===== CITATIONS =====>>
[[ { "id":"Wurm2011-ho"} ]]
<<===== CITATIONS =====<<

>>==== CSL ====>>

<?xml version="1.0" encoding="utf-8"?>
<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="display-and-sort" page-range-format="chicago">
  <info>
    <title>Chicago Manual of Style 17th edition (author-date)</title>
    <id>http://www.zotero.org/styles/chicago-author-date</id>
    <link href="http://www.zotero.org/styles/chicago-author-date" rel="self"/>
    <link href="http://www.chicagomanualofstyle.org/tools_citationguide.html" rel="documentation"/>
    <author>
      <name>Julian Onions</name>
      <email>julian.onions@gmail.com</email>
    </author>
    <contributor>
      <name>Sebastian Karcher</name>
    </contributor>
    <contributor>
      <name>Richard Karnesky</name>
      <email>karnesky+zotero@gmail.com</email>
      <uri>http://arc.nucapt.northwestern.edu/Richard_Karnesky</uri>
    </contributor>
    <contributor>
      <name>Andrew Dunning</name>
      <email>andrew.dunning@utoronto.ca</email>
      <uri>https://orcid.org/0000-0003-0464-5036</uri>
    </contributor>
    <contributor>
      <name>Matthew Roth</name>
      <email>matthew.g.roth@yale.edu</email>
      <uri> https://orcid.org/0000-0001-7902-6331</uri>
    </contributor>
    <contributor>
      <name>Brenton M. Wiernik</name>
    </contributor>
    <category citation-format="author-date"/>
    <category field="generic-base"/>
    <summary>The author-date variant of the Chicago style</summary>
    <updated>2018-01-24T12:00:00+00:00</updated>
    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
  </info>
  <locale xml:lang="en">
    <terms>
      <term name="editor" form="verb-short">ed.</term>
      <term name="container-author" form="verb">by</term>
      <term name="translator" form="verb-short">trans.</term>
      <term name="editortranslator" form="verb">edited and translated by</term>
      <term name="translator" form="short">trans.</term>
    </terms>
  </locale>
  <locale xml:lang="pt-PT">
    <terms>
      <term name="accessed">acedido a</term>
    </terms>
  </locale>
  <locale xml:lang="pt">
    <terms>
      <term name="editor" form="verb">editado por</term>
      <term name="editor" form="verb-short">ed.</term>
      <term name="container-author" form="verb">por</term>
      <term name="translator" form="verb-short">traduzido por</term>
      <term name="translator" form="short">trad.</term>
      <term name="editortranslator" form="verb">editado e traduzido por</term>
      <term name="and">e</term>
      <term name="no date" form="long">s.d</term>
      <term name="no date" form="short">s.d.</term>
      <term name="in">em</term>
      <term name="at">em</term>
      <term name="by">por</term>
      <!-- PUNCTUATION -->
      <term name="open-quote">"</term>
      <term name="close-quote">"</term>
    </terms>
  </locale>
  <macro name="secondary-contributors">
    <choose>
      <if type="chapter entry-dictionary entry-encyclopedia paper-conference" match="none">
        <group delimiter=". ">
          <names variable="editor translator" delimiter=". ">
            <label form="verb" text-case="capitalize-first" suffix=" "/>
            <name and="text" delimiter=", "/>
          </names>
          <names variable="director" delimiter=". ">
            <label form="verb" text-case="capitalize-first" suffix=" "/>
            <name and="text" delimiter=", "/>
          </names>
        </group>
      </if>
    </choose>
  </macro>
  <macro name="container-contributors">
    <choose>
      <if type="chapter entry-dictionary entry-encyclopedia paper-conference" match="any">
        <group prefix=", " delimiter=", ">
          <names variable="container-author" delimiter=", ">
            <label form="verb" suffix=" "/>
            <name and="text" delimiter=", "/>
          </names>
          <names variable="editor translator" delimiter=", ">
            <label form="verb" suffix=" "/>
            <name and="text" delimiter=", "/>
          </names>
        </group>
      </if>
    </choose>
  </macro>
  <macro name="editor">
    <names variable="editor">
      <name name-as-sort-order="first" and="text" sort-separator=", " delimiter=", " delimiter-precedes-last="always"/>
      <label form="short" prefix=", "/>
    </names>
  </macro>
  <macro name="translator">
    <names variable="translator">
      <name name-as-sort-order="first" and="text" sort-separator=", " delimiter=", " delimiter-precedes-last="always"/>
      <label form="short" prefix=", "/>
    </names>
  </macro>
  <macro name="recipient">
    <choose>
      <if type="personal_communication">
        <choose>
          <if variable="genre">
            <text variable="genre" text-case="capitalize-first"/>
          </if>
          <else>
            <text term="letter" text-case="capitalize-first"/>
          </else>
        </choose>
      </if>
    </choose>
    <names variable="recipient" delimiter=", ">
      <label form="verb" prefix=" " text-case="lowercase" suffix=" "/>
      <name and="text" delimiter=", "/>
    </names>
  </macro>
  <macro name="substitute-title">
    <choose>
      <if type="article-magazine article-newspaper review review-book" match="any">
        <text macro="container-title"/>
      </if>
    </choose>
  </macro>
  <macro name="contributors">
    <group delimiter=". ">
      <names variable="author">
        <name and="text" name-as-sort-order="first" sort-separator=", " delimiter=", " delimiter-precedes-last="always"/>
        <label form="short" prefix=", "/>
        <substitute>
          <names variable="editor"/>
          <names variable="translator"/>
          <names variable="director"/>
          <text macro="substitute-title"/>
          <text macro="title"/>
        </substitute>
      </names>
      <text macro="recipient"/>
    </group>
  </macro>
  <macro name="contributors-short">
    <names variable="author">
      <name form="short" and="text" delimiter=", " initialize-with=". "/>
      <substitute>
        <names variable="editor"/>
        <names variable="translator"/>
        <names variable="director"/>
        <text macro="substitute-title"/>
        <text macro="title"/>
      </substitute>
    </names>
  </macro>
  <macro name="interviewer">
    <names variable="interviewer" delimiter=", ">
      <label form="verb" prefix=" " text-case="capitalize-first" suffix=" "/>
      <name and="text" delimiter=", "/>
    </names>
  </macro>
  <macro name="archive">
    <group delimiter=". ">
      <text variable="archive_location" text-case="capitalize-first"/>
      <text variable="archive"/>
      <text variable="archive-place"/>
    </group>
  </macro>
  <macro name="access">
    <group delimiter=". ">
      <choose>
        <if type="graphic report" match="any">
          <text macro="archive"/>
        </if>
        <else-if type="article-journal bill book chapter legal_case legislation motion_picture paper-conference" match="none">
          <text macro="archive"/>
        </else-if>
      </choose>
      <choose>
        <if type="webpage post-weblog" match="any">
          <date variable="issued" form="text"/>
        </if>
      </choose>
      <choose>
        <if variable="issued" match="none">
          <group delimiter=" ">
            <text term="accessed" text-case="capitalize-first"/>
            <date variable="accessed" form="text"/>
          </group>
        </if>
      </choose>
      <choose>
        <if type="legal_case" match="none">
          <choose>
            <if variable="DOI">
              <text variable="DOI" prefix="https://doi.org/"/>
            </if>
            <else>
              <text variable="URL"/>
            </else>
          </choose>
        </if>
      </choose>
    </group>
  </macro>
  <macro name="title">
    <choose>
      <if variable="title" match="none">
        <choose>
          <if type="personal_communication" match="none">
            <text variable="genre" text-case="capitalize-first"/>
          </if>
        </choose>
      </if>
      <else-if type="bill book graphic legislation motion_picture song" match="any">
        <text variable="title" text-case="title" font-style="italic"/>
        <group prefix=" (" suffix=")" delimiter=" ">
          <text term="version"/>
          <text variable="version"/>
        </group>
      </else-if>
      <else-if variable="reviewed-author">
        <choose>
          <if variable="reviewed-title">
            <group delimiter=". ">
              <text variable="title" text-case="title" quotes="true"/>
              <group delimiter=", ">
                <text variable="reviewed-title" text-case="title" font-style="italic" prefix="Review of "/>
                <names variable="reviewed-author">
                  <label form="verb-short" text-case="lowercase" suffix=" "/>
                  <name and="text" delimiter=", "/>
                </names>
              </group>
            </group>
          </if>
          <else>
            <group delimiter=", ">
              <text variable="title" text-case="title" font-style="italic" prefix="Review of "/>
              <names variable="reviewed-author">
                <label form="verb-short" text-case="lowercase" suffix=" "/>
                <name and="text" delimiter=", "/>
              </names>
            </group>
          </else>
        </choose>
      </else-if>
      <else-if type="legal_case interview patent" match="any">
        <text variable="title"/>
      </else-if>
      <else>
        <text variable="title" text-case="title" quotes="true"/>
      </else>
    </choose>
  </macro>
  <macro name="edition">
    <choose>
      <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
        <choose>
          <if is-numeric="edition">
            <group delimiter=" " prefix=". ">
              <number variable="edition" form="ordinal"/>
              <text term="edition" form="short" strip-periods="true"/>
            </group>
          </if>
          <else>
            <text variable="edition" text-case="capitalize-first" prefix=". "/>
          </else>
        </choose>
      </if>
      <else-if type="chapter entry-dictionary entry-encyclopedia paper-conference" match="any">
        <choose>
          <if is-numeric="edition">
            <group delimiter=" " prefix=", ">
              <number variable="edition" form="ordinal"/>
              <text term="edition" form="short"/>
            </group>
          </if>
          <else>
            <text variable="edition" prefix=", "/>
          </else>
        </choose>
      </else-if>
    </choose>
  </macro>
  <macro name="locators">
    <choose>
      <if type="article-journal">
        <choose>
          <if variable="volume">
            <text variable="volume" prefix=" "/>
            <group prefix=" (" suffix=")">
              <choose>
                <if variable="issue">
                  <text variable="issue"/>
                </if>
                <else>
                  <date variable="issued">
                    <date-part name="month"/>
                  </date>
                </else>
              </choose>
            </group>
          </if>
          <else-if variable="issue">
            <group delimiter=" " prefix=", ">
              <text term="issue" form="short"/>
              <text variable="issue"/>
              <date variable="issued" prefix="(" suffix=")">
                <date-part name="month"/>
              </date>
            </group>
          </else-if>
          <else>
            <date variable="issued" prefix=", ">
              <date-part name="month"/>
            </date>
          </else>
        </choose>
      </if>
      <else-if type="legal_case">
        <text variable="volume" prefix=", "/>
        <text variable="container-title" prefix=" "/>
        <text variable="page" prefix=" "/>
      </else-if>
      <else-if type="bill book graphic legal_case legislation motion_picture report song" match="any">
        <group prefix=". " delimiter=". ">
          <group>
            <text term="volume" form="short" text-case="capitalize-first" suffix=" "/>
            <number variable="volume" form="numeric"/>
          </group>
          <group>
            <number variable="number-of-volumes" form="numeric"/>
            <text term="volume" form="short" prefix=" " plural="true"/>
          </group>
        </group>
      </else-if>
      <else-if type="chapter entry-dictionary entry-encyclopedia paper-conference" match="any">
        <choose>
          <if variable="page" match="none">
            <group prefix=". ">
              <text term="volume" form="short" text-case="capitalize-first" suffix=" "/>
              <number variable="volume" form="numeric"/>
            </group>
          </if>
        </choose>
      </else-if>
    </choose>
  </macro>
  <macro name="locators-chapter">
    <choose>
      <if type="chapter entry-dictionary entry-encyclopedia paper-conference" match="any">
        <choose>
          <if variable="page">
            <group prefix=", ">
              <text variable="volume" suffix=":"/>
              <text variable="page"/>
            </group>
          </if>
        </choose>
      </if>
    </choose>
  </macro>
  <macro name="locators-article">
    <choose>
      <if type="article-newspaper">
        <group prefix=", " delimiter=", ">
          <group delimiter=" ">
            <text variable="edition"/>
            <text term="edition"/>
          </group>
          <group>
            <text term="section" form="short" suffix=" "/>
            <text variable="section"/>
          </group>
        </group>
      </if>
      <else-if type="article-journal">
        <choose>
          <if variable="volume issue" match="any">
            <text variable="page" prefix=": "/>
          </if>
          <else>
            <text variable="page" prefix=", "/>
          </else>
        </choose>
      </else-if>
    </choose>
  </macro>
  <macro name="point-locators">
    <choose>
      <if variable="locator">
        <choose>
          <if locator="page" match="none">
            <choose>
              <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
                <choose>
                  <if variable="volume">
                    <group>
                      <text term="volume" form="short" suffix=" "/>
                      <number variable="volume" form="numeric"/>
                      <label variable="locator" form="short" prefix=", " suffix=" "/>
                    </group>
                  </if>
                  <else>
                    <label variable="locator" form="short" suffix=" "/>
                  </else>
                </choose>
              </if>
              <else>
                <label variable="locator" form="short" suffix=" "/>
              </else>
            </choose>
          </if>
          <else-if type="bill book graphic legal_case legislation motion_picture report song" match="any">
            <number variable="volume" form="numeric" suffix=":"/>
          </else-if>
        </choose>
        <text variable="locator"/>
      </if>
    </choose>
  </macro>
  <macro name="container-prefix">
    <text term="in" text-case="capitalize-first"/>
  </macro>
  <macro name="container-title">
    <choose>
      <if type="chapter entry-dictionary entry-encyclopedia paper-conference" match="any">
        <text macro="container-prefix" suffix=" "/>
      </if>
    </choose>
    <choose>
      <if type="webpage">
        <text variable="container-title" text-case="title"/>
      </if>
      <else-if type="legal_case" match="none">
        <group delimiter=" ">
          <text variable="container-title" text-case="title" font-style="italic"/>
          <choose>
            <if type="post-weblog">
              <text value="(blog)"/>
            </if>
          </choose>
        </group>
      </else-if>
    </choose>
  </macro>
  <macro name="publisher">
    <group delimiter=": ">
      <text variable="publisher-place"/>
      <text variable="publisher"/>
    </group>
  </macro>
  <macro name="date">
    <choose>
      <if variable="issued">
        <group delimiter=" ">
          <date variable="original-date" form="text" date-parts="year" prefix="(" suffix=")"/>
          <date variable="issued">
            <date-part name="year"/>
          </date>
        </group>
      </if>
      <else-if variable="status">
        <text variable="status" text-case="capitalize-first"/>
      </else-if>
      <else>
        <text term="no date" form="short"/>
      </else>
    </choose>
  </macro>
  <macro name="date-in-text">
    <choose>
      <if variable="issued">
        <group delimiter=" ">
          <date variable="original-date" form="text" date-parts="year" prefix="[" suffix="]"/>
          <date variable="issued">
            <date-part name="year"/>
          </date>
        </group>
      </if>
      <else-if variable="status">
        <text variable="status"/>
      </else-if>
      <else>
        <text term="no date" form="short"/>
      </else>
    </choose>
  </macro>
  <macro name="day-month">
    <date variable="issued">
      <date-part name="month"/>
      <date-part name="day" prefix=" "/>
    </date>
  </macro>
  <macro name="collection-title">
    <choose>
      <if match="none" type="article-journal">
        <choose>
          <if match="none" is-numeric="collection-number">
            <group delimiter=", ">
              <text variable="collection-title" text-case="title"/>
              <text variable="collection-number"/>
            </group>
          </if>
          <else>
            <group delimiter=" ">
              <text variable="collection-title" text-case="title"/>
              <text variable="collection-number"/>
            </group>
          </else>
        </choose>
      </if>
    </choose>
  </macro>
  <macro name="collection-title-journal">
    <choose>
      <if type="article-journal">
        <group delimiter=" ">
          <text variable="collection-title"/>
          <text variable="collection-number"/>
        </group>
      </if>
    </choose>
  </macro>
  <macro name="event">
    <group delimiter=" ">
      <choose>
        <if variable="genre">
          <text term="presented at"/>
        </if>
        <else>
          <text term="presented at" text-case="capitalize-first"/>
        </else>
      </choose>
      <text variable="event"/>
    </group>
  </macro>
  <macro name="description">
    <choose>
      <if variable="interviewer" type="interview" match="any">
        <group delimiter=". ">
          <text macro="interviewer"/>
          <text variable="medium" text-case="capitalize-first"/>
        </group>
      </if>
      <else-if type="patent">
        <group delimiter=" " prefix=". ">
          <text variable="authority"/>
          <text variable="number"/>
        </group>
      </else-if>
      <else>
        <text variable="medium" text-case="capitalize-first" prefix=". "/>
      </else>
    </choose>
    <choose>
      <if variable="title" match="none"/>
      <else-if type="thesis personal_communication speech" match="any"/>
      <else>
        <group delimiter=" " prefix=". ">
          <text variable="genre" text-case="capitalize-first"/>
          <choose>
            <if type="report">
              <text variable="number"/>
            </if>
          </choose>
        </group>
      </else>
    </choose>
  </macro>
  <macro name="issue">
    <choose>
      <if type="legal_case">
        <text variable="authority" prefix=". "/>
      </if>
      <else-if type="speech">
        <group prefix=". " delimiter=", ">
          <group delimiter=" ">
            <text variable="genre" text-case="capitalize-first"/>
            <text macro="event"/>
          </group>
          <text variable="event-place"/>
          <text macro="day-month"/>
        </group>
      </else-if>
      <else-if type="article-newspaper article-magazine personal_communication" match="any">
        <date variable="issued" form="text" prefix=", "/>
      </else-if>
      <else-if type="patent">
        <group delimiter=", " prefix=", ">
          <group delimiter=" ">
            <!--Needs Localization-->
            <text value="filed"/>
            <date variable="submitted" form="text"/>
          </group>
          <group delimiter=" ">
            <choose>
              <if variable="issued submitted" match="all">
                <text term="and"/>
              </if>
            </choose>
            <!--Needs Localization-->
            <text value="issued"/>
            <date variable="issued" form="text"/>
          </group>
        </group>
      </else-if>
      <else-if type="article-journal" match="any"/>
      <else>
        <group prefix=". " delimiter=", ">
          <choose>
            <if type="thesis">
              <text variable="genre" text-case="capitalize-first"/>
            </if>
          </choose>
          <text macro="publisher"/>
        </group>
      </else>
    </choose>
  </macro>
  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" givenname-disambiguation-rule="primary-name" collapse="year" after-collapse-delimiter="; ">
    <layout prefix="(" suffix=")" delimiter="; ">
      <group delimiter=", ">
        <choose>
          <if variable="issued accessed" match="any">
            <group delimiter=" ">
              <text macro="contributors-short"/>
              <text macro="date-in-text"/>
            </group>
          </if>
          <!---comma before forthcoming and n.d.-->
          <else>
            <group delimiter=", ">
              <text macro="contributors-short"/>
              <text macro="date-in-text"/>
            </group>
          </else>
        </choose>
        <text macro="point-locators"/>
      </group>
    </layout>
  </citation>
  <bibliography hanging-indent="true" et-al-min="11" et-al-use-first="7" subsequent-author-substitute="&#8212;&#8212;&#8212;" entry-spacing="0">
    <sort>
      <key macro="contributors"/>
      <key variable="issued"/>
      <key variable="title"/>
    </sort>
    <layout suffix=".">
      <group delimiter=". ">
        <text macro="contributors"/>
        <text macro="date"/>
        <text macro="title"/>
      </group>
      <text macro="description"/>
      <text macro="secondary-contributors" prefix=". "/>
      <text macro="container-title" prefix=". "/>
      <text macro="container-contributors"/>
      <text macro="edition"/>
      <text macro="locators-chapter"/>
      <text macro="collection-title-journal" prefix=", " suffix=", "/>
      <text macro="locators"/>
      <text macro="collection-title" prefix=". "/>
      <text macro="issue"/>
      <text macro="locators-article"/>
      <text macro="access" prefix=". "/>
    </layout>
  </bibliography>
</style>
<<==== CSL ====<<

>>==== INPUT ====>>
[
  {
    "author": [
      {
        "family": "Wurm",
        "given": "Klaus"
      }
    ],
    "collection-title": "Quellen und Studien zur Philosophie",
    "edition": "Reprint 2011",
    "id": "Wurm2011-ho",
    "issued": {
      "date-parts": [
        [
          2011
        ]
      ]
    },
    "keyword": "!!! Plotinus translation",
    "publisher": "De Gruyter",
    "publisher-place": "Berlin",
    "title": "Substanz und Qualität : Ein Beitrag zur Interpretation der plotinischen Traktate VI,1, 2 und 3",
    "type": "book"
  }
]
<<==== INPUT ====<<



>>===== VERSION =====>>
1.0
<<===== VERSION =====<<
```
we get the result
```html
<div class="csl-bib-body">
  <div class="csl-entry">Wurm, Klaus. 2011. <i>Substanz Und Qualität : Ein Beitrag Zur Interpretation Der Plotinischen Traktate VI,1, 2 Und 3</i>. Reprint 2011. Quellen Und Studien Zur Philosophie. Berlin: De Gruyter.</div>
</div>
```

Pinging @njbart who has been helpful before in setting me straight about related issues.

Note: the biblatex translation code in pandoc is sensitive to the `language` field, but ONLY for the "title" (not for the journal title, etc.).  Maybe that is wrong.

Note 2: there is nothing in CSL corresponding to the `language` field here, as I recall, so once we have translated from bibtex the information is lost, I think. (But I dimly remember something about this coming up before; hopefully @njbart can help.)

**Comment by njbart** [comment]
TLDR: Use `langid ={de}` in biblatex data in order to prevent case changes in the various title fields. (Other non-English `langid` tags will have the same effect, of course.)

Background: Confusingly, the direct counterpart of CSL’s `language` in biblatex is `langid`.

biblatex’s `language`, by contrast, is not used for flagging metadata processing options but can be used (but only in conjunction with a biblatex processor) to trigger the printing of language information in a bibliography entry. One typical use case I am aware of is is indicating the languages of bi- or multilingual texts, e.g. “Greek and English“, and the like.

**Comment by njbart** [comment]
As to journal titles, I think they’re assumed to be unchangeable, capitalization-wise, even in English, and IIRC, pandoc has implemented that rule.

**Comment by jgm** [comment]
If journal titles are assumed to be unchangeable, then it seems that citeproc's current behavior (where it is title-casing the journal title) is wrong even for English titles -- correct?

Pandoc's conversion from bibtex/biblatex, by contrast, seems okay (given the usual bibtex convention that capital letters not protected with `{}` are treated as lowercase).  Solution for @RobertGoulding is to protect the capital letters in the journal title.

@njbart does that seem like a correct assessment?

**Comment by njbart** [comment]
> If journal titles are assumed to be unchangeable, then it seems that citeproc's current behavior (where it is title-casing the journal title) is wrong even for English titles -- correct?

In your example (“we get the result ...”), an entry is formatted according to instructions from a CSL style file. If this file contains a `title-case` instruction (which it does), title-casing is correct.

What I was talking about is the conversion `-f biblatex -t markdown -s`. Here, pandoc converts the various biblatex title fields to sentence case, as expected by CSL – with the exception of `journal` and `journaltitle` fields, which are not normally protected by curly braces in biblatex data and which are not converted to sentence case by biblatex (the program) either when using sentence-cases styles such as `biblatex-apa`.

> Solution for @RobertGoulding is to protect the capital letters in the journal title.

I continue to think the solution is to add `langid = {de}` to this entry.

**Comment by jgm** [comment]
> Here, pandoc converts the various biblatex title fields to sentence case, as expected by CSL – with the exception of journal and journaltitle fields, which are not normally protected by curly braces in biblatex data and which are not converted to sentence case by biblatex (the program) either when using sentence-cases styles such as biblatex-apa.

So, we should not do any case conversion for journal and journaltitle fields.  That is one thing to change, then.

> > Solution for @RobertGoulding is to protect the capital letters in the journal title.
>
> I continue to think the solution is to add `langid = {de}` to this entry.

To make sure I understand you correctly:  this should affect pandoc's conversion from bibtex to CSL JSON, not by changing how the journal and journaltitle are converted (since by the above logic they should be left alone in any case), but by adding a `language` field, which citeproc should then be sensitive to? Is that correct?

**Comment by njbart** [comment]
> So, we should not do any case conversion for journal and journaltitle fields. That is one thing to change, then.

No, that’s already in place in pandoc/citeproc.

> So, we should not do any case conversion for journal and journaltitle fields. That is one thing to change, then.

No, pandoc/citeproc already convert (`-f biblatex -t markdown -s`) biblatex `langid = {de}` to markdown aka CSL YAML `language: de`.

It’s just that the OP should add appropriate `langid` tags to biblatex entries, at the very least to all of the non-English ones, though it’d be good practice to add them to every entry. In fact , the latter is necessary in case the biblatex data are to be processed by biblatex (the program) as well.

**Comment by jgm** [comment]
> > So, we should not do any case conversion for journal and journaltitle fields. That is one thing to change, then.
>
> No, that’s already in place in pandoc/citeproc.

But see above, my test of `pandoc sample.bib -s -t native`:

```
                       , ( "collection-title"
                         , MetaInlines
                             [ Str "Quellen"
                             , Space
                             , Str "und"
                             , Space
                             , Str "studien"
                             , Space
                             , Str "zur"
                             , Space
                             , Str "philosophie"
                             ]
                         )
```
sample.bib contains "Quellen und Studien zur Philosophie", so there is a change in case here.

**Comment by njbart** [comment]
But that’s `collection-title` (= biblatex `series`), not `container-title` (= biblatex `journal`).

**Comment by RobertGoulding** [comment]
My original question had nothing to do with journal title, only
series title.

Robert Goulding

**Comment by jgm** [comment]
OK, right.  So, is pandoc handling the collection-title conversion from bibtex's series correctly?

@RobertGoulding - can you try what @njbart suggests and add `langid = {de}` to your entry?

**Comment by cagix** [comment]
@jgm @njbart The advice about `langid = {de}` solves my (related) problem with mixed case titles. Double curly braces were not a solution in this case, as Pandoc would wrap the title in a `[...]{.nocase}` (conversion from Markdown to Markdown). Thank you very much!

**Comment by njbart** [comment]
> So, is pandoc handling the collection-title conversion from bibtex's series correctly?

In non-English entries: yes. (No case conversion at all.)

In English entries: currently converted to sentence case. I tend to think that’s ok since, unlike `journal`, I’m not aware of any universally accepted convention to keep series titles unchanged (i.e., in title case), and I’m not aware of any bug reports relating to this so far either.
